NC-3341: Add Turnstile isExpired documentation (#10117)
cloudflare/cloudflare-docs: content/turnstile/get-started/client-side-rendering.md

## Access a widget's state

> In addition to the `render()` function, Turnstile supports obtaining the widget's response from a `widgetId` via the `turnstile.getResponse(widgetId: string)` function. If the `widgetId` is omitted `turnstile.getResponse` returns the response from the last widget created.
> After some time a widget may have become expired and needs to be refreshed (i.e. by calling `turnstile.reset(widgetId: string)`). In this event `turnstile.getResponse` will still return the last response, however, the response will no longer be valid, because it has expired.
> Whether a widget expired can be checked by either subscribing to the `expired-callback` (see below) or using the `turnstile.isExpired(widgetId: string)` function, which returns true if the widget is expired. If the `widgetId` is omitted `turnstile.isExpired` returns whether the last widget created is expired or not.

Reviewer: ```suggestion
In addition to the `render()` function, Turnstile supports obtaining the widget's response from a `widgetId` via the `turnstile.getResponse(widgetId: string)` function. If you omit the `widgetId`, `turnstile.getResponse()` returns the response from the last created widget.

After some time, a widget may become expired and needs to be refreshed (by calling `turnstile.reset(widgetId: string)`). If a widget has expired, `turnstile.getResponse()` will still return the last response, but the response will no longer be valid because it has expired.

You can check if a widget has expired by either subscribing to the `expired-callback` or using the `turnstile.isExpired(widgetId: string)` function, which returns `true` if the widget is expired. If you omit `widgetId`, `turnstile.isExpired()` returns whether the last created widget is expired or not.
```
